app/menu.js

```diff
@@ -312,6 +313,28 @@ module.exports = function createMenu({createWindow, updatePlugins}) {
       },
       {
         role: 'togglefullscreen'
+      },
+      {
+        id: 'ENTER_QUICK_FULL_SCREEN',
+        label: 'Enter Quick Full Screen',
+        accelerator: 'Cmd+Enter',
+        visible: true,
+        click(item, focusedWindow) {
+          if (focusedWindow) {
+            focusedWindow.rpc.emit('enter quick full screen req');
+          }
+        }
+      },
+      {
+        id: 'LEAVE_QUICK_FULL_SCREEN',
+        label: 'Leave Quick Full Screen',
+        accelerator: 'Cmd+Enter',
```

Comment:
@flybayer Any thoughts on what the accelerator should be? :thinking: `Alt+Enter`?